index.html

```diff
@@ -913,6 +926,10 @@ <h2 class="h4">
       <p class="text-secondary">(no tracks)</p>
     <% } %>
   <% } %>
+  <div class="row mb-3">
+    <label for="edit_note-txt" googl="true" style="padding: 0;"><strong>Edit note:</strong></label>
+    <textarea width="50%" rows="8" name="edit-note" id="edit-note-txt" googl="true"><%= editNote %></textarea>  
```

Comment:
The `width` attribute is unnecessary here, since contents of a bootstrap row fill the whole width by default.

Please use the bootstrap styles so the text area border is consistent: add `class="form-control"`. If you're interested in limiting the width of the text area, or placing it beside the label, please reference the [Bootstrap Form Layout documentation](https://getbootstrap.com/docs/5.1/forms/layout/)

The contents of the `<textarea>` element needs to have HTML escaping applied, to prevent a cross-site HTML injection attack. Here's a link which shows this: https://reubot.github.io/magicisrc/?mbid=bdde4d50-0cb4-4bdb-9870-71c738bea97e&edit-note=%3C%2Ftextarea%3E%3Ch1%3ECross+site+HTML+injection+attack%21%3C%2Fh1%3E (scroll down to the bottom of the page). Note that this escaping must not have the `\n` to `<br>` translation applied, since `<br>` elements are not permitted in a text area.